ros_buildfarm/templates/status/blocked_releases_page_job.xml.em

```diff
@@ -30,7 +30,7 @@
     refspec=None,
 ))@
   <scmCheckoutRetryCount>2</scmCheckoutRetryCount>
-  <assignedNode>agent_on_master||buildagent</assignedNode>
+  <assignedNode>agent_on_jenkins || agent_on_master || buildagent</assignedNode>
```

Comment:
Have you confirmed that this whitespace is stripped before the string evaluation?

Reply:
What string evaluation are you referring to? The empy template process, Jenkins label expressions, or something else?

Reply:
The agent matching was what I was referring to.